### PR TITLE
New command option '-check'

### PR DESCRIPTION
**Pull request by avancinirodrigo**
- [x] #1412 
- [x] #1460 
- [x] #1461

### CODE REVIEW

File: test/commands.lua [hunk 1]
```diff
@@ -103,7 +103,9 @@ basic = {
 	scriptdir           = {arg = packageInfo().path},
 	scriptnofile        = {arg = "abcd1234.lua"},
 	scriptnoluafile     = {arg = filePath("agents.csv")},
-	tmp                 = {script = "tmp.lua"}
+	tmp                 = {script = "tmp.lua"},
+	luacheck1			= {script = "luacheck.lua", arg = "-strict"},
+	luacheck2			= {script = "luacheck.lua", arg = "-debug"}
```
Comment (pedro-andrade-inpe) [inline test/commands.lua]: Just check indentation here.

Comment (pedro-andrade-inpe) [inline test/commands.lua]: You will also need to add the two log files related to these tests.

Comment (avancinirodrigo) [inline test/commands.lua]: I used tab, is it a space?

Comment (pedro-andrade-inpe) [inline test/commands.lua]: After luacheck it is space, before it is tab.

File: test/commands.lua [hunk 2]
```diff
@@ -104,8 +104,8 @@ basic = {
 	scriptnofile        = {arg = "abcd1234.lua"},
 	scriptnoluafile     = {arg = filePath("agents.csv")},
 	tmp                 = {script = "tmp.lua"},
-	luacheck1			= {script = "luacheck.lua", arg = "-strict"},
-	luacheck2			= {script = "luacheck.lua", arg = "-debug"}
+	luacheck1           = {script = "luacheck1.lua", arg = "-strict"},
```
Comment (pedro-andrade-inpe) [inline test/commands.lua]: It seems that script should be "luacheck.lua" in both cases.

Comment (avancinirodrigo) [inline test/commands.lua]: Ops!

**Comment by avancinirodrigo** [comment]
#1463

**Pull request merged**